content/en/docs/reference/command-line-tools-reference/kube-proxy.md

```diff
@@ -116,7 +116,7 @@ kube-proxy [flags]
 <td colspan="2">--feature-gates mapStringBool</td>
 </tr>
 <tr>
```

Comment:
Pls leave this page alone. We will update it when the new release is out.

Reply:
Fixed